soccer/NewRefereeModule.cpp

```diff
@@ -179,6 +179,13 @@ void NewRefereeModule::run() {
         ballPlacementx = packet->wrapper.designated_position().x();
         ballPlacementy = packet->wrapper.designated_position().y();
 
+        std::string blue_name = blue_info.name;
+        for(char& letter : blue_name){
+            letter=tolower(letter);
+        }
+
+        blueTeam(blue_name == "robojackets");
```

Comment:
An important thing to note about checking this way is that if the name is not set in the refbox or is incorrect we will always be Yellow. I am not actually sure if that is a good thing or not.